src/modules/organization/event/components/table/Table.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="flex justify-between my-2">
+    <div class="flex space-x-3">
+      <BaseSearch class="h-4" :placeholder="'Search'" @search="handleSearch" />
+      <BaseButton
+        class="flex justify-center items-center // h-4 px-2 space-x-1 flex-shrink-0"
+        @click="approveAllRequest"
+      >
+        <div>Approved all</div>
+        <base-icon class="w-3 h-3">
+          <check-circle-icon />
+        </base-icon>
+      </BaseButton>
+    </div>
+
+    <SingleNameSelect
+      :optionNames="sortBy"
+      :optionValues="sortByVal"
+      v-model="sortOption"
+    />
+  </div>
+  <div class="p-3 bg-white // rounded-2xl // w-full">
+    <table class="table-auto // w-full">
+      <thead>
+        <tr>
+          <th
+            v-for="headerKey in headerKeys"
+            :key="headerKey"
+            class="pb-1 // border-b-2 border-blue-11 // text-left"
+          >
+            {{ header[headerKey] }}
```

Comment:
what is headerKey and why we need to iterate headerKey instead just iterate header

Reply:
this is header 
<img width="338" alt="image" src="https://user-images.githubusercontent.com/37561295/116742551-b5763e00-aa21-11eb-9631-fece4b631406.png">

Reply:
headerKey is an array of the key of headers. In this case, it would be ['name', 'email', 'phoneNumber','status']

Reply:
Fixed to use Object.entries